docs/sources/administration/image_rendering.md

```diff
@@ -43,7 +45,62 @@ For further information and instructions refer to the [plugin details](https://g
 
 The [Grafana image renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) can also be run as a remote HTTP rendering service. In this setup Grafana will render an image by making a HTTP request to the remote rendering service, which in turn render the image and returns it back in the HTTP response to Grafana.
 
-To configure Grafana to use a remote HTTP rendering service, please refer to [rendering](/installation/configuration/#rendering) configuration section.
+As an alternative to installing and running the image renderer as a plugin you can run it as a remote image rendering service running as a node.js application or inside a Docker container.
+
+**Running as standalone node.js application:**
```

Comment:
I would move `Using Docker` section above this section since Docker should be the preferred/first thing to use when considering remote HTTP rendering service.